### ISSUE TITLE
The table of contents should manifest on top of the text, rather than pushing the text over, on mobile

### ISSUE THREAD

**Issue by david-christiansen** [leanprover/reference-manual#139]
On mobile devices, opening the table of contents should pop up a menu _over_ the content, rather than pushing the content to the right and sometimes reflowing text:

![Screenshot_20241104-113837](https://github.com/user-attachments/assets/4716c543-5c18-4ad3-a90f-9634f5bad814)
![Screenshot_20241104-113849](https://github.com/user-attachments/assets/772180a4-8e7c-4d91-a341-7fd0cb453bea)

**Comment by jakobvase** [comment]
I would drop the transition that makes it slide in. It is a thing of wonder and beauty, and it looks strange - the text in the toc reflows when you click the button. Also, animations are annoying for some people. Go for simplicity.

**Comment by jakobvase** [comment]
Once the transition is removed, it's easier to make it float on top of the text, because you don't have to deal with the transforms:

```css

#toc {
    background-color: var(--verso-toc-background-color);
    color: var(--verso-toc-text-color);
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    width: 0;
}

#toc:has(#toggle-toc:checked) {
    width: 15em;
}

@media screen and (max-width: 700px) {
    #toc {
        position: fixed;
        z-index: 10;
    }
}
```

**Comment by david-christiansen** [comment]
Fixed in #148. Thanks!
